applications/content-drafting.md

```diff
@@ -0,0 +1,29 @@
+Machine translation allows to quickly draft content in natural language. Applications:
+
+# Mock-up localization
+Raw machine-translated text is a good mock-up to verify how the target language appears in localized content such as:
+* user interface,
+* images and diagrams.
+
+MT produces natural-language content that shall not be verified for accuracy at this step, but presents such features as target alphabet and typical word/sentence length. 
+When applied to localized content, MT allows to quickly issues, for example:
+* national characters corrupted or missing,
+* words or phrases truncated due to lack of screen space,
+* wrong sort order.
+Detecting these issues at early phase allows taking corrective actions, in parallel with the actual professional translation or post-editing, or planning for necessary post-processing of localized content, for example:
+* enable more character sets in localization (locale, font, etc.)
```

Comment:
Since this [list](https://machinetranslate.org/style#lists) has conjugated verbs, each item should start with uppercase and end with a period.

Reply:
Done